Client/src/python/dbs/apis/dbsClient.py

```diff
@@ -317,6 +317,22 @@ def insertBulkBlock(self, blockDump):
         :type blockDump: dict
 
         """
+        isEventCount = True
+        for f in blockDump['files']:
+            if not isEventCount: break
+            for fl in f['file_lumi_list']:
+                if 'event_count' in fl:
+                    if fl['event_count'] is None:
+                        isEventCount = False
+                        break
+                else:
+                    isEventCount = False
+                    break
+        if not isEventCount:
```

Comment:
I just change the name to sync with above. But you can keep whatever name you want. I just thought it might be better to have flag without 'not' operator